utility.py

```
@@ -62,25 +69,24 @@ def _get_config_option(section: str, option: str, is_bool: bool = False, *, conf
         if is_bool:
             return config.getboolean(section, option)
-
         return config.get(section, option)
 
     except configparser.NoSectionError as e:
         # Check config template for the section if it doesn't exist in user's config
         if config != config_template:
             return _get_config_option(section, option, is_bool, config=config_template)
         else:
-            raise RuntimeError(f"Invalid config section '{section}': {e}")
+            raise e
```

Ruff (0.14.0)

Use `raise` without specifying exception name

Remove exception name

(TRY201)